client/dashboard/sites/settings-php/index.tsx

```diff
@@ -41,7 +41,15 @@ export default function PHPVersionSettings( { siteSlug }: { siteSlug: string } )
 
 	if ( ! canUpdate ) {
 		return (
-			<PageLayout size="small" header={ <SettingsPageHeader title="PHP" /> }>
+			<PageLayout
+				size="small"
+				header={
+					<SettingsPageHeader
+						title="PHP"
+						description={ __( 'Sites on the Personal plan run on our recommended PHP version.' ) }
```

Comment:
The `Personal` is not supposed to be hardcoded here. 🙈

Reply:
Couldn't be that easy.

Reply:
p1749028363285189/1749028308.668089-slack-CRWCHQGUB 🏃‍♂️

Reply:
Added in https://github.com/Automattic/wp-calypso/pull/103981/commits/f72e6308cad930ca68cfad93090f836c0ce363cf?